File: java/src/tk/csparpa/gofdp/factorymethod/ConcreteCreator.java
package tk.csparpa.gofdp.factorymethod;

/**
 * 
 * Implementation of abstract factory class and method. Modify it as you need!
 *
 */

public class ConcreteCreator extends AbstractCreator {

	@Override
	public AbstractProduct factoryMethod() {
		System.out.println("Called: implementation of factory method; "
				+ "will return: ConcreteProduct instance");
		return new ConcreteProduct();
	}
}

File: python/factorymethod/concrete_creator.py
from abstract_creator import AbstractCreator
from concrete_product import ConcreteProduct 

class ConcreteCreator(AbstractCreator):

    """
    Implementation of abstract factory class and method. Modify it as you need!
    """
    
    def factory_method(self):
        print  "Called: implementation of factory method; will return: ConcreteProduct instance"
        return ConcreteProduct()